moshi/src/main/java/com/squareup/moshi/JsonAdapter.java

```diff
@@ -30,6 +30,11 @@
 
 /**
  * Converts Java values to JSON, and JSON values to Java.
+ *
+ * JsonAdapter instances provided by Moshi are thread-safe, meaning multiple threads can safely use a single instance
```

Comment:
Missing `<p>` tags to separate paragraphs.

Reply:
Fixed